noir-projects/aztec-nr/aztec/src/note/note_getter.nr

```diff
@@ -146,14 +145,12 @@ where
     // to fail.
     let filter_fn = options.filter;
     let filter_args = options.filter_args;
-    let filtered_notes = filter_fn(opt_notes, filter_args);
+    let notes = filter_fn(retrieved_notes, filter_args);
 
-    let notes = array::collapse(filtered_notes);
```

Comment:
With these changes `array::collapse` is now completely unused in aztec-packages.